### DISCUSSION TITLE
If a rogue organization owner removes me, will GitHub Support help?

### DISCUSSION THREAD

**Discussion by Christian-Toney** [community/community#23081] [New to GitHub]
<p>Hi!</p>
<p>I want to appoint another owner on my GitHub organization so that they may add other members to it, but I recognize that other organization owners have the same access as the person who created the organization.</p>
<p>When I created the organization, I designated it as owned by a business/institution instead of my personal account. Does this mean if I legally own the business that owns the GitHub organization, then GitHub Support may help me if I am wrongly removed from the organization?</p>
<p>Best regards,<br>
Christian</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/sudobeast">@Sudobeast</a> and welcome :tada:,</p>
<p>Sounds like your organization is set up under our corporate Terms of Service, if so, we do have ways to attempt to regain ownership if the situation arises in which you are removed, but it is not a guarantee. This of course varies on a situation by situation basis.</p>
<p>Hope this helps :smile:</p>
<p>Lili</p>

**Reply by mehraUTD** [reply]
@queenofcorgis can yo please help gain access back to my own organization?
https://github.com/UTDClassroomOrg .. please help! this is urgent and I accidentally removed myself. only owner

**Reply by queenofcorgis** [reply]
@mehraUTD: 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket once submitted, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

**Comment by billythekid** [comment]
<p>Just in case it’s helpful for people in future, here’s some more information about our Corporate Terms of Service, where ownership of an organization account can be attributed to the company instead of the individual owners:</p>
<p><a href="https://docs.github.com/github/site-policy/github-corporate-terms-of-service">GitHub Corporate Terms of Service</a></p>
<p><a href="https://docs.github.com/organizations/managing-organization-settings/upgrading-to-the-corporate-terms-of-service">Upgrading to the Corporate Terms of Service</a></p>
<p>:cowboy_hat_face:</p>
